packages/mco/components/modals/app-manage-policies/helper/assign-policy-view-footer.tsx

```diff
@@ -22,9 +30,31 @@ import { DRPolicyType } from '../utils/types';
 import '../../../../style.scss';
 import '../style.scss';
 
+export const isValidLabelInput = (
+  labels: string[] = [],
+  // Flag to Skip labels length validation
+  defaultValue: boolean
+): boolean =>
+  labels.length
+    ? labels.every((label) => {
+        const requirement = requirementFromString(label);
+        // "key" or "key=value" is supported
+        return (
+          !!requirement &&
+          [Operator.Equals, Operator.Exists].includes(
+            requirement?.operator as Operator
```

Comment:
```suggestion
            requirement?.operator
```
Update `isValidLabelInput` to return `operator` as Operator

Reply:
@GowthamShanmugam can we fix this?

Reply:
Here the return type is boolean. this condition checks if a particular operator exists.

Reply:
i got the confusion with defaultValue, let me remove that.